packages/prisma/selects/booking.ts

```diff
@@ -0,0 +1,11 @@
+import { Prisma } from "@prisma/client";
+
+export const bookingMinimalSelect = Prisma.validator<Prisma.BookingSelect>()({
+  id: true,
+  title: true,
+  description: true,
+  customInputs: true,
+  startTime: true,
+  endTime: true,
+  attendees: true,
+});
```

Comment:
I find "reusing common selects" doesn't work in practise @zomars - because it's a symptom rather than the problem itself - when a select overlaps so much you can actually re-use the type you typically can abstract to a library function. That makes this a half-way-there-fix but not the actual solution.

Reply:
Agreed, just think it's a nice way to make it obvious that this is not the only place where this selects will be used.